README.md

```diff
@@ -76,6 +76,34 @@ import requests
 requests.post('http://grounding.indra.bio/ground', json={'text': 'kras'})
 ```
 
+## Resource usage
+Gilda loads grounding terms into memory when first used. If memory usage
+is an issue, the following options are recommended.
+
+1. Run a single instance of Gilda as a local web service that one or more
+other processes send requests to.
+
+2. Create a custom Grounder instance that only loads a subset of terms
+approrpiate for a narrow use case.
+
+3. Gilda also offers an optional sqlite back-end which significantly decreases
+memory usage and results in minor drop in the number of strings grounder per
+unit time. The sqlite back-end database can be built as follows with an
+optional `[db_path]` argument, which if used, should use the .db extension. If
+not specified, the .db file is generated in Gilda's default resource folder.
+
+```bash
+python -m gilda.resources.sqlite_adapter [db_path]
+```
+
+A Grounder instance can then be instantiated as follows:
+
+```python
+from gilda.grounder import Grounder
+gr = Grounder(db_path)
```

Comment:
So to generate the sqlite database if need and use it, it'd be like:

```py
import gilda, subprocess
from gilda.grounder import Grounder
db_path = pathlib.Path(f"{gilda.resources.resource_dir}/grounding_terms.db")
if not db_path.exists():
    subprocess.run("python -m gilda.resources.sqlite_adapter")
grounder = Grounder(db_path)
```

Reply:
Yes, doing it that way is possible, though if multiple processes do this at the same time, can lead to conflicts. So overall better to prepare the DB before runtime.